R/checkStuff.R

```diff
@@ -66,6 +66,15 @@ checkStuff = function(fun, par.set, design, learner, control) {
     assertNumber(control$target.fun.value, na.ok = FALSE)
   }
 
+  if (length(control$save.on.disk.at) > 0L && (control$iters + 1) %nin% control$save.on.disk.at)
```

Comment:
I thought checkStuff is for checks that can only be done during the runtime of mbo whilst all the other stuff should be checked directly in the setMBOControl methods?